translator/translate/metrics/config/registered_metrics.go

```diff
@@ -52,7 +54,8 @@ var Registered_Metrics_Windows = map[string][]string{
 }
 
 var DisableWinPerfCounters = map[string]bool{
-	"statsd":     true,
-	"procstat":   true,
-	"nvidia_smi": true,
+	"statsd":      true,
+	"procstat":    true,
+	"nvidia_smi":  true,
+	common.JmxKey: true,
```

Comment:
this is basically saying we only support jmx on linux right?

Reply:
We support jmx on all 3 platforms

Reply:
But we are adding jmx to the `DisableWinPerfCounters` map, meaning that we cannot have jmx running with windows performance? Not sure i understand

Reply:
But we are adding jmx to the DisableWinPerfCounters map, meaning that we cannot have jmx running with windows performance? Not sure i understand

Reply:
This means if should not go into the conf file